components/browser/menu2/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- Empty by default, allows others to theme as they see fit -->
```

Comment:
We have some other components that use styles to allow the app to set a colour here. The [BrowserTabsTray](https://github.com/mozilla-mobile/android-components/blob/40d97f22a8639849297c8bf6099ed23871a3cdbc/components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/BrowserTabsTray.kt#L43) is an example off the top of my mind you could check out.

Reply:
I'm going to remove this file anyways, it only matters for the other PR.